lib/screens/v2/candidate_generator/gl_eink.ex

```diff
@@ -207,6 +207,9 @@ defmodule Screens.V2.CandidateGenerator.GlEink do
 
       {:ok, departures} ->
         {:ok, departures}
+
+      :error ->
```

Comment:
A potential suggestion: could we show the headsign here so the riders see something when this failure happens? Evaluating the headsign shouldn't need a request so we should always have that info. What do you think?

Reply:
I think that would make sense since this code is specifically for GL! Ok if it's done in a separate "Add headway mode to v2 GL e-ink screens" task, to let this fix get out faster?

Reply:
Yep, sounds good!